docs/ingestion/native-batch-input-source.md

```diff
@@ -773,7 +773,7 @@ The following is an example of a Combining input source spec:
         "delegates" : [
          {
           "type": "local",
-          "filter" : "*.csv",
+          "filter" : "**.csv",
```

Comment:
This is not really true though. The local input source filter continues to work the old way, right? 

I think that's also where the contention lies. We can't change the filter for the local input source as that has been in use for a while. We do want to have similar `filter` behavior for other input sources too. I think the only way out is to have a different `globFilter` in addition to `filter`. The latter applies just to the filer names.

Reply:
Can you help show me how did the local input source do it? Maybe I can use the same library?

Reply:
These?
```
import org.apache.commons.io.filefilter.TrueFileFilter;
import org.apache.commons.io.filefilter.WildcardFileFilter;
```

Reply:
Ah, because it is used in combination with
```
    final Collection<File> files = FileUtils.listFiles(
        Preconditions.checkNotNull(this.baseDir, "baseDir").getAbsoluteFile(),
        new WildcardFileFilter(filter),
        TrueFileFilter.INSTANCE
    );
```

But blob store is a fake filesystem it doesn't have a real folder structure.

Reply:
If the syntax difference is what's concerning, then let's just have `globFilter` (to call out the different syntax) for blob store ingestion and that's it.  I don't think `filter` is needed.